File: android/shared/src/main/java/com/horizend/crmdelivery/shared/SupportedLanguage.kt
package com.horizend.crmdelivery.shared

enum class SupportedLanguage(val code: String, val label: String, val isRtl: Boolean = false) {
    English("en", "English"),
    German("de", "Deutsch"),
    Turkish("tr", "Türkçe"),
    Arabic("ar", "العربية", isRtl = true),
    Russian("ru", "Русский")
}

File: apple/shared/Sources/SupportedLanguage.swift
import Foundation

public enum SupportedLanguage: String, CaseIterable, Identifiable {
    case english = "en"
    case german = "de"
    case turkish = "tr"
    case arabic = "ar"
    case russian = "ru"

    public var id: String { rawValue }

    public var label: String {
        switch self {
        case .english: return "English"
        case .german: return "Deutsch"
        case .turkish: return "Türkçe"
        case .arabic: return "العربية"
        case .russian: return "Русский"
        }
    }

    public var isRightToLeft: Bool {
        self == .arabic
    }
}
